doc/vba_functions.html

```diff
@@ -500,7 +500,7 @@
 	q Quarter <br>
 	m Month <br>
 	y Day of year <br>
```

Comment:
@lucboudreau, yes it is. See it here: https://msdn.microsoft.com/en-us/library/b5xbyt6f%28v=vs.90%29.aspx